general/cuda.hpp

```diff
@@ -63,9 +63,15 @@ void* CuMemAlloc(void **d_ptr, size_t bytes);
 /// Allocates managed device memory
 void* CuMallocManaged(void **d_ptr, size_t bytes);
 
+/// Allocate page-locked (pinned) host memory
+void* CuMemAllocHost(void **ptr, size_t bytes);
+
 /// Frees device memory and returns destination ptr.
 void* CuMemFree(void *d_ptr);
 
+/// Frees page-locked (pinned) host memory and returns destination ptr.
+void* CuMemFreeHost(void *ptr);
```

Comment:
Same here, I'd call this `CuMemFreePinned` or `CuMemFreeHostPinned`.

Reply:
Done